openmetadata-service/src/main/java/org/openmetadata/service/resources/alerts/AlertActionResource.java

```diff
@@ -172,9 +172,9 @@ public ResultList<AlertAction> listAlertAction(
   @Valid
   @Operation(
       operationId = "getAlertActionByID",
-      summary = "Get a alert alert action",
+      summary = "Get an alert action by id",
       tags = "alertAction",
```

Comment:
Please use `Id` instead of `id` everywhere.

Reply:
Updated the id to use "Id" in all documentation usages.